dialogue-target/src/main/java/com/palantir/dialogue/BinaryRequestBody.java

```diff
@@ -23,15 +23,11 @@
 import java.io.InputStream;
 import java.io.OutputStream;
 
-/**
- * Streamed binary response data with Content-Type <code>application/octet-stream</code>.
- */
+/** Streamed binary response data with Content-Type <code>application/octet-stream</code>. */
 public interface BinaryRequestBody extends Closeable {
 
-    /**
-     * Invoked to write data to the response stream. Called exactly once.
-     */
-    void write(OutputStream responseBody) throws IOException;
+    /** Invoked to write data to the request stream. Called exactly once. */
```

Comment:
```suggestion
    /** Invoked to write data to the request stream. */
```

If we're limited and no space exists on the queue, or the request is cancelled, this may be called at all, but that should be okay since this is closeable, and we do close the BinaryRequestBody in those scenarios.